src/ninja.cc

```diff
@@ -845,6 +846,76 @@ int NinjaMain::ToolInputs(const Options* options, int argc, char* argv[]) {
   return 0;
 }
 
+int NinjaMain::ToolMultiInputs(const Options* options, int argc, char* argv[]) {
+  // The inputs tool uses getopt, and expects argv[0] to contain the name of
+  // the tool, i.e. "inputs".
+  argc++;
+  argv--;
+
+  bool print0 = false;
+
+  optind = 1;
+  int opt;
+  const char* defaultfieldseparator = "\t";
+  const char* fieldseparator = defaultfieldseparator;
```

Comment:
the `defaultfieldseparator` variable doesn't seem to be useful here. Also please use snake_case for naming variables, they'll be much more readable :-)